scripts/ci/install.unit

```diff
@@ -5,7 +5,7 @@ set -x # echo commands
 
 # install NPM dependencies
 pushd bokehjs
-npm install --no-save --no-progress phantomjs
+npm install --no-save --no-progress phantomjs typescript
```

Comment:
This should install typescript version as specified in package.json. Otherwise, in the window between new version is out and we update, this particular test job may fail compilation.

Reply:
The syntax is `typescript@3.3.1` (for the currently used version).

Reply:
`npm install` doesn't use package.json?

Reply:
I definitely don't want the burden of manually keeping that in sync. I'd just change to `npm install` (with no arguments) instead

Reply:
`npm install` does use package.json, but `npm install package_name` does not.

Reply:
of course it does....

Reply:
That's new to me. I'm pretty sure it didn't work like this before.

Reply:
Sorry, I was being sarcastic and complaining that of course it behaves in this inconsistent way re: ignoring package.json in some cases but not others

Reply:
One thing to keep in mind is that `--no-save` is important here, because otherwise npm would overwrite package.json. One it will make a mess, but more importantly it actually changes version specifications, so in our case it would replace `3.3.1`, with `^3.3.1`, which as minor it is, it could lead to issues.

Reply:
noted, thanks